packages/emdb/src/util/fetchJSON.js

```diff
@@ -3,6 +3,16 @@
 const crossFetch = require('cross-fetch');
 
 module.exports = async function fetchJSONBrowser(url) {
-  const result = await crossFetch(url);
+  const controller = new AbortController();
+
+  // 5 second timeout:
+  const timeoutId = setTimeout(() => controller.abort(), 300000);
```

Comment:
You can use `AbortSignal.timeout` instead: https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout